Dart.gitignore

```diff
@@ -11,11 +11,9 @@ pubspec.lock
 # If you don't generate documentation locally you can remove this line.
 doc/api/
 
-# Avoid committing generated Javascript files:
-*.dart.js
-*.info.json      # Produced by the --dump-info flag.
-*.js             # When generated by dart2js. Don't specify *.js if your
-                 # project includes source files written in JavaScript.
-*.js_
-*.js.deps
-*.js.map
+# Directories and files created by other development environments 
```

Comment:
These rules below are covered by existing templates:

 - `Global/JetBrains.gitignore`
 - `Global/macOS.gitignore`

I'm 👍 to the cleanup here, but I need to prevent duplicating rules covered elsewhere because they're a maintenance burden I can't support currently.

Reply:
The above `gitignore`s might not be relevant to someone developing a flutter app and if I'm developing on a windows device then having to add the MacOS `gitignore` seems like a very unnecessary nuance.

And I believe it can be a good way to just allow the developer to add the dart ignore and be able to manage the same

Reply:
TIL https://github.com/github/gitignore/blob/master/Global/JetBrains.gitignore – Nice!

We should train devs to learn to global ignore their OS- and tool-specific bits IMHO.

There is disagreement here, though!

Reply:
Let me if I should just remove the same or they would be fine in the setup

Reply:
> Let me if I should just remove the same or they would be fine in the setup

Yes please. The other templates will get rule changes that are not synchronized here, so for the moment this is my current recommendation.